infrastructure/cli/integration/api_login_test.go

```diff
@@ -0,0 +1,47 @@
+package integration
+
+import (
+	"os/exec"
+	"testing"
+
+	"reflect"
+)
+
+func TestApiLogin(t *testing.T) {
```

Comment:
the idea here is to have one file per command (so I split the existing tests).

There's still some code that can be extracted in functions (especially the way we run the binaries... but I prefer to do that in a follow up pr)